qiskit/test/mock/backends/fake_100q/conf_100q.json

```diff
@@ -0,0 +1,68 @@
+{
+    "acquisition_latency": [],
+    "backend_name": "100q",
+    "backend_version": "0.0.0",
+    "basis_gates": [
+        "id",
+        "u1",
+        "u2",
+        "u3"
+    ],
+    "conditional": true,
+    "conditional_latency": [],
+    "coupling_map": [
+        [
+            0,
+            1
+        ]
+    ],
+    "discriminators": [],
+    "dt": 1.3333e-09,
+    "dtm": 1.0500000000000001e-08,
+    "gates": [
+        {
+            "name": "TODO",
+            "parameters": [],
+            "qasm_def": "TODO"
+        }
+    ],
+    "hamiltonian": {
+        "description": "",
+        "h_str": [],
+        "qub": {},
+        "vars": {}
+    },
+    "local": true,
+    "max_shots": 65536,
+    "meas_kernels": [],
+    "meas_level": [],
+    "meas_levels": [
+        0,
+        1,
+        2
+    ],
+    "meas_lo_range": [],
+    "meas_map": [
```

Comment:
There must be an entry for every qubit, eg., every qubit index must live in the meas_map.